py/mpconfig.h

```diff
@@ -490,6 +490,11 @@
 #define MICROPY_LONGINT_IMPL (MICROPY_LONGINT_IMPL_NONE)
 #endif
 
+// Support for calls to buildin.pow() with 3 integer arguments
+#ifndef MICROPY_INT_POW3
+#define MICROPY_INT_POW3 (1)
```

Comment:
I set the default to 1 because this is a standard feature of Python and it seems that the default setting should not break the spec. Also, the default behaviour before my patch had this functionality enabled; the issue was simply that the old implementation crashed on non-trival tests if you had multi-precision enabled. The new code is only present when you already have an entire MPZ library linked in. If maintainers seriously need to save space a much better option is to remove MPZ support.

I'm not going to fight over this but I really do think that the default should be to follow the Python spec.